### PR TITLE
The value of !set command support multi line

### PR DESCRIPTION
**Pull request by xiedeyantu**

### CODE REVIEW

**Comment by xiedeyantu** [comment]
@julianhyde Could you help me to have a look this pr?

File: src/main/java/net/hydromatic/quidem/Quidem.java [hunk 1]
```diff
@@ -611,6 +611,9 @@ private Command nextCommand() throws IOException {
             String[] parts = line.split(" ");
             String propertyName = parts[1];
             String valueString = parts[2];
+            while (valueString.startsWith("\"") && !valueString.endsWith("\"")) {
```
Comment (mihaibudiu) [inline src/main/java/net/hydromatic/quidem/Quidem.java:614]: does this work for 3 lines?

Comment (xiedeyantu) [reply src/main/java/net/hydromatic/quidem/Quidem.java:614]: Yes, by identifying the beginning and ending double quotes.

Comment (julianhyde) [reply src/main/java/net/hydromatic/quidem/Quidem.java:614]: Add a test with multiple lines and I'll believe you. :)

**Comment by julianhyde** [comment]
Can you add tests for more than 2 lines, and tests where some of the lines are empty, and tests where line contains Quidem comment syntax.

Also update the documentation in README.md.

File: src/main/java/net/hydromatic/quidem/Quidem.java (hunk 1 above)
Comment (xiedeyantu) [reply src/main/java/net/hydromatic/quidem/Quidem.java:614]: Sorry, it really doesn't work. I will adjust it. Thank you for your review.

Comment (xiedeyantu) [reply src/main/java/net/hydromatic/quidem/Quidem.java:614]: I have modified the test cases and readme.

**Comment by xiedeyantu** [comment]
@julianhyde I have modified the test cases and readme.Please have a look again thanks!

**Review by mihaibudiu — APPROVED** [review]

**Comment by julianhyde** [comment]
Thanks for the PR! I have merged (with a few revisions).

When you have successfully used these extensions in a Calcite PR (using 0.12-SNAPSHOT), let me know, and I will release Quidem version 0.12.

**Comment by xiedeyantu** [comment]
> Thanks for the PR! I have merged (with a few revisions).
> 
> When you have successfully used these extensions in a Calcite PR (using 0.12-SNAPSHOT), let me know, and I will release Quidem version 0.12.

OK, I will finish this task as soon as possible!

**Comment by xiedeyantu** [comment]
@julianhyde I have tested it locally and it works.
